contracts/governance/compatibility/GovernorCompatibilityBravo.sol

```diff
@@ -100,10 +100,10 @@ abstract contract GovernorCompatibilityBravo is IGovernorTimelock, IGovernorComp
     }
 
     function cancel(uint256 proposalId) public virtual override(IGovernor, Governor) {
-        ProposalDetails storage details = _proposalDetails[proposalId];
+        address proposer = _proposalDetails[proposalId].proposer;
```

Comment:
I plan to remove that in 5.0

Cannot remove it earlier, because in an upgrade from 4.8 to 4.9 we would lose the proposer info, that is not in the core. 

We could do a fallback lookup in the core, and only write/read from here if the core doesn't have the data ... is that worth it for 4.9?

Reply:
The code complexity is not too much, what is complicated is managing the differences between vanilla and upgradeable contracts if we don't want the fallback logic to be present in the vanilla contracts.

Reply:
So do we want it?